### DISCUSSION TITLE
List classes under one screen size declaration.

### DISCUSSION THREAD

**Discussion by ianstreator** [tailwindlabs/tailwindcss#10103] [Ideas]
( Current syntax )
![tailwind-current](https://user-images.githubusercontent.com/20363735/208195021-a25ee4f5-8b78-4404-bb91-fad22ada21f3.PNG)
( Proposed syntax )
![tailwind-new](https://user-images.githubusercontent.com/20363735/208195129-0d79d77b-7a06-4cfd-a196-64ebbd0e5743.PNG)

**Comment by wongjn** [comment]
[Here's a response to a similar suggestion](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314
